r/aoa_script.R

```diff
@@ -17,6 +16,7 @@ library(rjson) #rjson for reading json input job file
 library(raster) #raster-Package for working with various raster formats
 library(gdalcubes) #gdalcubes-Package for creating, handling and using spatio-temporal datacubes
 library(kernlab) #kernlab for training kernel based support vector machines
+library(testthat) #testthat for tests 
```

Comment:
Muss dies auch in der `Dockerfile` Datei drin sein? Bzw. hast du dieses Paket installiert?

Reply:
Ach ja :O wollte ich gertern noch erwähnt haben. Aber ja das müsste auch dort mit rein.